### ISSUE TITLE
Use template as SWC feature?

### ISSUE THREAD

**Issue by mistergreen** [BowlerHatLLC/feathers#1577]
I was thinking, wouldn't it be easier to style if feathers use rasters UI from a vector SWC?
Designers would design or mod a UI easily in Illustrator or Flash pro.

**Comment by joshtynjala** [comment]
Do you mean that you'd like to have a way to provide a set of vector assets, and then Feathers will automatically convert them to BitmapData and create texture atlases for you at runtime?

**Comment by joshtynjala** [comment]
It is also worth mentioning that all of the example themes in Feathers are designed as vectors in Animate CC. They are exported as texture atlases using Animate's "Generate Sprite Sheet" feature. Feathers doesn't create the texture atlas at runtime, but the designer (me, in this case) was able to create the skins in Animate.

You can find the FLA files for the example themes here: https://feathersui.com/help/theme-sources.html

**Comment by mistergreen** [comment]
> Do you mean that you'd like to have a way to provide a set of vector assets, and then Feathers will automatically convert them to BitmapData and create texture atlases for you at runtime?

Yes. The scale of the asset isn't essential since they're vector. I think you can label the vector symbols with some 'class' so you can generate a sprite sheet on runtime. Can you also affect the styles of the SWC on runtime? I'm not sure if this has any benefit over what is currently implemented.
